admin_guide/manage_nodes.adoc

```diff
@@ -241,7 +241,7 @@ controller] can be evacuated; the replication controllers create new pods on
 other nodes and remove the existing pods from the specified node(s). Bare pods,
 meaning those not backed by a replication controller, are unaffected by default.
 
-To evacuate one or more nodes:
+To evacuate all or selected pods on one or more nodes:
 
 ----
 $ oc adm drain <node1> <node2>
```

Comment:
We are saying 'evacuate all or selected pods' but we are not telling how to do that using a command.
We should talk about `--pod-selector` here and show that as optional param to the command.
```
 $ oc adm drain <node1> <node2> [--pod-selector=<pod_selector>]
```
I've assumed '[ ]' is used for optional flags if that is not the case use what ever convention is used in other places.